pyronear/datasets/wildfire/fire_labeler.py

```diff
@@ -1,10 +1,13 @@
+#!usr/bin/python
```

Comment:
Why do you add the shebang ? this file is not intended to be used as a script

Reply:
same remark for all new shebang in the current PR

Reply:
Oops, always copy pasted it with the encoding, you are correct!